### PR TITLE
update model files, docs, and tests

### PR DESCRIPTION
**Pull request by edward-leardi-ibm**
Addresses, #19, #20, and #21

### CODE REVIEW

File: core/model.py [hunk 1]
```diff
@@ -124,12 +124,22 @@ def _predict(self, args):
...
     def _post_process(self, predictions):
-        if predictions.shape[1] == 12:  # multivariate model
+        if predictions.shape[1] == 15:  # multivariate model
             predictions = np.array(predictions)
-            predictions[:, 0] = np.clip(predictions[:, 0], 0, 10)  # HOURLYVISIBILITY - should be 0-10
+            predictions[:, 0] = np.clip(np.rint(predictions[:, 0]), 0, 10)  # HOURLYVISIBILITY - should be int 0-10
+            predictions[:, 1] = np.clip(predictions[:, 1], -15, 106) # HOURLYDRYBULBTEMPF - should be between extremes recorded in nyc
+            predictions[:, 2] = np.clip(predictions[:, 2], -13, 104) # HOURLYWETBULBTEMPF - should be between extremes recorded in nyc
+            predictions[:, 3] = np.clip(predictions[:, 3], -30, 95) # HOURLYDewPointTempF - should be between extremes recorded in nyc
+            predictions[:, 4] = np.clip(predictions[:, 4], 0, 101) # HOURLYRelativeHumidity - should be between extremes recorded in nyc
```
Comment (bdwyer2) [inline core/model.py]: flake8
```suggestion
            predictions[:, 1] = np.clip(predictions[:, 1], -15, 106)  # HOURLYDRYBULBTEMPF - should be between extremes recorded in nyc
            predictions[:, 2] = np.clip(predictions[:, 2], -13, 104)  # HOURLYWETBULBTEMPF - should be between extremes recorded in nyc
            predictions[:, 3] = np.clip(predictions[:, 3], -30, 95)  # HOURLYDewPointTempF - should be between extremes recorded in nyc
            predictions[:, 4] = np.clip(predictions[:, 4], 0, 101)  # HOURLYRelativeHumidity - should be between extremes recorded in nyc
```

File: core/model.py [hunk 2]
```diff
@@ -124,12 +124,22 @@ def _predict(self, args):
...
-            predictions[:, 10] = np.maximum(predictions[:, 10], 0)  # HOURLYPrecip - should be non-negative
+            predictions[:, 6] = np.clip(predictions[:, 6], 28, 31) # HOURLYStationPressure
+            predictions[:, 7] = np.clip(predictions[:, 7], 28, 31) # HOURLYSeaLevelPressure
+            predictions[:, 8] = np.maximum(predictions[:, 8], 0)  # HOURLYPrecip - should be non-negative
+            predictions[:, 9] = np.clip(predictions[:, 9], 28, 31) # HOURLYAltimeterSetting
+            predictions[:, 10] = np.clip(predictions[:, 10], -1, 1) # HOURLYWindDirectionSin - should be -1 to 1
+            predictions[:, 11] = np.clip(predictions[:, 11], -1, 1) # HOURLYWindDirectionCos - should be -1 to 1
+            predictions[:, 12] = np.clip(np.rint(predictions[:, 12]), 0, 1) # HOURLYPressureTendencyIncr - should be 0 or 1
+            predictions[:, 13] = np.clip(np.rint(predictions[:, 13]), 0, 1) # HOURLYPressureTendencyDecr - should be 0 or 1
+            predictions[:, 14] = np.clip(np.rint(predictions[:, 14]), 0, 1) # HOURLYPressureTendencyCons - should be 0 or 1
```
Comment (bdwyer2) [inline core/model.py]:
```suggestion
            predictions[:, 6] = np.clip(predictions[:, 6], 28, 31)  # HOURLYStationPressure
            predictions[:, 7] = np.clip(predictions[:, 7], 28, 31)  # HOURLYSeaLevelPressure
            predictions[:, 8] = np.maximum(predictions[:, 8], 0)  # HOURLYPrecip - should be non-negative
            predictions[:, 9] = np.clip(predictions[:, 9], 28, 31)  # HOURLYAltimeterSetting
            predictions[:, 10] = np.clip(predictions[:, 10], -1, 1)  # HOURLYWindDirectionSin - should be -1 to 1
            predictions[:, 11] = np.clip(predictions[:, 11], -1, 1)  # HOURLYWindDirectionCos - should be -1 to 1
            predictions[:, 12] = np.clip(np.rint(predictions[:, 12]), 0, 1)  # HOURLYPressureTendencyIncr - should be 0 or 1
            predictions[:, 13] = np.clip(np.rint(predictions[:, 13]), 0, 1)  # HOURLYPressureTendencyDecr - should be 0 or 1
            predictions[:, 14] = np.clip(np.rint(predictions[:, 14]), 0, 1)  # HOURLYPressureTendencyCons - should be 0 or 1
```

File: tests/test.py [hunk 3]
```diff
@@ -83,32 +83,36 @@ def test_predict():
...
+        assert 31 > prediction[6] > 28  # HOURLYStationPressure
+        assert 31 > prediction[7] > 28  # HOURLYSeaLevelPressure
+        assert 3 > prediction[8] >= 0  # HOURLYPrecip
+        assert 31 > prediction[9] > 28  # HOURLYAltimeterSetting
+        assert abs(prediction[9] - prediction[7]) < 0.1  # HOURLYAltimeterSetting ~= HOURLYSeaLevelPressure
+        assert 1 > prediction[10] > -1 # HOURLYWindDirectionSin - should be -1 to 1
+        assert 1 > prediction[11] > -1 # HOURLYWindDirectionCos - should be -1 to 1
+        assert prediction[12] in [0,1] # HOURLYPressureTendencyIncr - should be 0 or 1
+        assert prediction[13] in [0,1] # HOURLYPressureTendencyDecr - should be 0 or 1
+        assert prediction[14] in [0,1] # HOURLYPressureTendencyCons - should be 0 or 1
```
Comment (bdwyer2) [inline tests/test.py]:
```suggestion
        assert 1 > prediction[10] > -1  # HOURLYWindDirectionSin - should be -1 to 1
        assert 1 > prediction[11] > -1  # HOURLYWindDirectionCos - should be -1 to 1
        assert prediction[12] in [0,1]  # HOURLYPressureTendencyIncr - should be 0 or 1
        assert prediction[13] in [0,1]  # HOURLYPressureTendencyDecr - should be 0 or 1
        assert prediction[14] in [0,1]  # HOURLYPressureTendencyCons - should be 0 or 1
```

File: tests/test.py [hunk 4]
```diff
@@ -83,32 +83,36 @@ def test_predict():
...
 
     for prediction in ms_json['predictions']:
         # predicted HOURLYDRYBULBTEMPF for the next 48 hours
-        assert 100 > max(prediction)
-        assert 0 < min(prediction)
+        assert 106 > max(prediction) # hottest recorded in nyc
+        assert -15 < min(prediction) # coldest recorded in nyc
```
Comment (bdwyer2) [inline tests/test.py]:
```suggestion
        assert 106 > max(prediction)  # hottest recorded in nyc
        assert -15 < min(prediction)  # coldest recorded in nyc
```

File: tests/test.py [hunk 5]
```diff
@@ -83,32 +83,36 @@ def test_predict():
...
         assert len(prediction) == 48
 
     for prediction in uv_json['predictions']:
         # predicted HOURLYDRYBULBTEMPF for the next hour
-        assert 100 > max(prediction)
-        assert 0 < min(prediction)
+        assert 106 > max(prediction) # hottest recorded in nyc
+        assert -15 < min(prediction) # coldest recorded in nyc
```
Comment (bdwyer2) [inline tests/test.py]:
```suggestion
        assert 106 > max(prediction)  # hottest recorded in nyc
        assert -15 < min(prediction)  # coldest recorded in nyc
```

File: core/model.py (hunk 1 above)
Comment (edward-leardi-ibm) [reply core/model.py]: thanks, I see it now in the travis job where it says `at least two spaces before inline comment`, didn't know about this rule

**Pull request merged**